### ISSUE TITLE
flavour screenshots are very big when a vanilla flavour is active

### ISSUE THREAD

**Issue by chriswmartin** [glitch-soc/mastodon#261]
if using a vanilla flavour the new flavour selector screenshots aren't scaled down to fit the page

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by marrus-sh** [comment]
Yeah I wasn't sure whether we wanted to modify vanilla's stylesheets to fix this or leave them exactly the same as upstream

**Comment by marrus-sh** [comment]
I tagged this as `incomplete idea` because I'm not sure if we've decided whether we want to modify vanilla styling to support static pages or not but the fix is pretty straightforward

I'm guessing that profile metadata is also super broken with vanilla styles so it might be something to do—we could just add a `glitch.scss` import for additional non-upstream addins.
